File: python/3342.py
a = 0
b = 0
n = int(input())

for i in range(n):
    i_eh_par = i % 2 == 0
    
    for j in range(n):
        j_eh_par = j % 2 == 0
        
        if (i_eh_par):
            if (j_eh_par):
                a += 1
            else:
                b += 1
        else:
            if (j_eh_par):
                b += 1
            else:
                a += 1

print('{} casas brancas e {} casas pretas'.format(a, b))

File: src/java/3342.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        int a = 0, b = 0, n;
        boolean i_eh_par, j_eh_par;
        Scanner input = new Scanner(System.in);
        
        n = input.nextInt();
        
        for (int i = 0; i < n; i++) {
            i_eh_par = i % 2 == 0;
        
            for (int j = 0; j < n; j++) {
                j_eh_par = j % 2 == 0;
                
                if (i_eh_par) {
                    if (j_eh_par) {
                        a++;
                    } else {
                        b++;
                    }
                } else {
                    if (j_eh_par) {
                        b++;
                    } else {
                        a++;
                    }
                }
            }
        }
    
        System.out.printf("%d casas brancas e %d casas pretas\n", a, b);
    }
}
